gulpfile.coffee

```diff
@@ -5,9 +5,9 @@ coffee = require('gulp-coffee')
 
 OPTIONS =
 	files:
-		coffee: [ 'lib/**/*.coffee', 'tests/**/*.spec.coffee', 'gulpfile.coffee' ]
+		coffee: [ 'lib/**/*.coffee', 'gulpfile.coffee' ]
 		app: 'lib/**/*.coffee'
-		tests: 'tests/**/*.spec.coffee'
+		tests: [ 'lib/**/*.coffee', 'tests/**/*.spec.js']
```

Comment:
LGTM except couldn't this line remain a string and just be `'tests/**/*.spec.js'` ?

Reply:
Indeed it can :+1: